cirq/google/common_serializers.py

```diff
@@ -218,3 +218,20 @@ def _near_mod_2(e, t, atol=1e-8):
                                              value_func=lambda _: 0.5),
         ]),
 ]
+"""CZ Serializer"""
+CZ_SERIALIZER = op_serializer.GateOpSerializer(
+    gate_type=ops.CZPowGate,
+    serialized_gate_id='cz',
+    args=[
+        op_serializer.SerializingArg(serialized_name='half_turns',
+                                     serialized_type=float,
+                                     gate_getter='exponent')
+    ])
+"""CZ Deserializer"""
```

Comment:
nit: I would suggest using a comment here instead of a string literal, since this isn't actually a docstring for anything. Could make that change in a separate PR since this would affect existing code in this module. Would also be nice to add whitespace between the serializer instances, though maybe the formatter will do that if we convert to comments.

Reply:
Done.